src/core/pluralize.js

```diff
@@ -58,9 +74,16 @@ function getPluralizer() {
   });
 
   // returns pluralized term if `text` needs pluralization, "" otherwise
-  return function pluralizeDfn(text) {
-    const normText = normalize(text.toLowerCase());
-    const plural = pluralOf(normText);
-    return links.has(plural) && !dfns.has(plural) ? plural : "";
+  return {
+    pluralizeDfn: text => {
+      const normText = normalize(text.toLowerCase());
+      const plural = pluralOf(normText);
+      return links.has(plural) && !dfns.has(plural) ? plural : "";
```

Comment:
Lets keep a single function and return an array of two items from it?

Reply:
Done, it seems I only ever needed to change 1 line.